manifest_dev.yml

```diff
@@ -1,8 +1,10 @@
 ---
-inherit: manifest_base.yml
+buildpack: staticfile_buildpack
+applications:
+  - name: proxy
+    memory: 64M
 routes:
-- route: fec-dev-proxy.app.cloud.gov
-#- route: transition.fec.gov
```

Comment:
Is transition.fec.gov no longer something we need to worry about, or just not for the `dev` environment?

Oh never mind, it was commented out anyway, so this is cleanup. :-)

Reply:
@ccostino yep, thanks!